spec/operations/books_lists/destroy_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'faker'
+
+describe ::BooksLists::Destroy do
+  subject(:creates_list) { ::BooksLists::Create.call(book:, list:) }
+  subject(:destroy_list) { ::BooksLists::Destroy.call(book:, list:) }
+
+  let(:book) { create(:book) }
+  let(:list) { create(:list) }
+
+  it 'destroy list' do
+    creates_list
```

Comment:
Inside your `it` statement should be only things which you are testing (expect bla bla bla). Other stuff you should move to let or before blocks